skfp/descriptors/constitutional.py

```diff
@@ -0,0 +1,252 @@
+from typing import Optional, Union
+
+from rdkit.Chem import AddHs, Mol
+from rdkit.Chem.Descriptors import MolWt
+from rdkit.Chem.rdMolDescriptors import CalcNumRings, CalcNumRotatableBonds
+
+
+def validate_molecule(func):
```

Comment:
1. Would fit better in `utils/validators`
2. Add typing: func is Callable, and this also returns a Callable